apps/backoffice-v2/src/domains/workflows/hooks/mutations/useUpdateDocumentByIdMutation/useUpdateDocumentByIdMutation.tsx

```diff
@@ -41,6 +41,10 @@ export const useUpdateDocumentByIdMutation = ({
       const previousWorkflow = queryClient.getQueryData(workflowById.queryKey);
 
       queryClient.setQueryData(workflowById.queryKey, (oldWorkflow: TWorkflowById) => {
+        if (!oldWorkflow) {
+          return;
+        }
```

Comment:
Trying to update an associated company document would error due to workflow being undefined.